_source/api/logzio-public-api.yml

```diff
@@ -3138,3 +3283,100 @@ definitions:
           format: int32
         description: For user access, `2`. For admin access, `3`.
         example: [2]
+
+  # ::::: Archive DEFINITIONS
+  ArchiveSettings:
+    type: object
+    properties:
+      storageType:
+        type: string
+        enum:
+          - S3
+          - BLOB
+        description: Specifies which settings are applicable - AWS S3 bucket or Azure Blob storage.
+        default: none
+        example: S3
+      enabled:
+        type: boolean
+        description: If `true`, archiving is currently enabled.
+        default: true
+        example: true
+      compressed:
+        type: boolean
+        description: If `true`, logs are compressed before they are archived.
+        default: true
```

Comment:
@alexserdukov I was sure we always archive compressed...isn't that the case? if so why do we give an option to the customer to change it